scrapd/core/apd.py

```diff
@@ -217,7 +217,27 @@ def parse_details_page_notes(details_page_notes):
             first_cap = index
             break
 
-    return squished[first_cap:]
+    # Remove HTML tags.
+    no_html = re.sub(re.compile('<.*?>'), '', squished[first_cap:])
+
+    # Remove tabs and, if subjects included, remove.
+    remove_subjects = re.split(r'\s{2,}', no_html)
```

Comment:
It seems fragile to rely on 2 spaces or more. Not sure what would be a better solution without using the HTML tags though 🤔

Reply:
Yes, I was having a hard time because the split on tabs didn't do much for many of the Notes, because the whitespace was a series of spaces instead 🙄